src/ocean/meta/codegen/CTFE.d

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+
+    Utilities for common CTFE operations needed for code generation
+
+    Copyright:
+        Copyright (c) 2017 sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module ocean.meta.codegen.CTFE;
+
+import ocean.meta.types.Qualifiers;
+
+/*******************************************************************************
+
+    Converts integer to string
+
+    Params:
+        i = integer to convert
+
+*******************************************************************************/
+
+istring toString ( long i )
```

Comment:
It is cleaned up copy of those. As I have answered previously to Gavin, intention it to deprecate all symbols in `ocean.core.Traits` in one go when replacement package is sufficiently battle tested, that will be both more reliable and easier migration-wise (one big sed script to run once).

For now only signed long version was copied as I expect other case be of unnecessary complexity in CTFE (How often do you format numbers > long.max in that context?). May copy second overload later.